.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
   "printWidth": 120,
-  "singleQuote": true,
+  "trailingComma": "none",
```

Comment:
Why disable it? It's really convenient to avoid having another change when you add a property to an object

Reply:
In general, I can relate to it, I like it better to - helps to reorder the lines without issues e.g.
Did it cause that's the standard we have right now in `monday-ui-react-core` and `monday-ui-style`
If you want, we can introduce a new standards